exercises/practice/phone-number/src/test/java/PhoneNumberTest.java

```diff
@@ -10,7 +10,7 @@ public class PhoneNumberTest {
     @Test
     public void cleansTheNumber() {
         String expectedNumber = "2234567890";
-        String actualNumber = new PhoneNumber("(223) 456-7890").getNumber();
+        String actualNumber = new PhoneNumber("(223) 456-7890").toString();
```

Comment:
If the tests are changed, the _proof_ or _example_ implementation in the `phone-number` folder also needs to be updated, otherwise the CI will fail to run.